src/session.ts

```diff
@@ -528,10 +531,6 @@ async function warmTransfer(
     target: WebPhoneSession,
     transferOptions: any = {}
 ): Promise<ReferClientContext> {
-    await (this.localHold ? Promise.resolve(null) : this.hold());
-
-    await delay(300);
-
```

Comment:
AFAIR `warmTransfer` requires that call should be on hold. Is it a safe fix? Has anything changed in telco?